content/id/contribute/_index.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
Did you mean: `content/id/docs/contribute/_index.md` ?

Reply:
Oh my, thanks for that! @sftim 
I've moved it to the right path.